README.md

```diff
@@ -7,114 +7,6 @@ Please find the sample that fits your use-case from the table below.
 | Sample | Description | Use-Case |
 |--------|-------------|----------|
 | [Okta-Hosted Login](/okta-hosted-login) | An application server that uses the hosted login page on your Okta org, then creates a cookie session for the user in the Spring application. | Traditional web applications with server-side rendered pages. |
-| [Custom Login Page](/custom-login) | An application server that uses the Okta Sign-In Widget on a custom login page within the application, then creates a cookie session for the user in the Spring application. | Traditional web applications with server-side rendered pages. |
+| [Custom Login Page](/custom-login) | An application server that uses the Okta Sign-In Widget on a custom login page within the application, then creates a cookie session for the user in the Spring application. | Traditional web applications having custom login page with server-side rendered pages. |
```

Comment:
having custom login page with server-side rendered pages. -> with a custom login page and server-side rendered pages.